src/main/scala/cognite/spark/v1/StringDataPointsRelation.scala

```diff
@@ -27,8 +29,41 @@ case class StringDataPointsFilter(
     externalId: Option[String]
 )
 
+case class StringDataPointsDelete(
```

Comment:
This looks identical to `NumericDataPointsDelete` so we can probably use just one `DataPointsDelete`.

Very cool that it works.